tests/test_unit.py

```diff
@@ -136,6 +136,50 @@ def test_reference_in_summary(self):
         self.assertEqual(lines_got, lines_want)
 
 
+    # Test for added xref coverage and third party xrefs staying as-is
+    def test_reference_in_summary_more_xrefs(self):
+        lines_got = """
+If a ~dateutil.time.stream() is attached to this download, then the downloaded
+resource will be written to the stream.
+
+Args:
+    transport (~google.cloud.requests.Session()): A ``requests`` object which can
+        make authenticated requests.
+
+    timeout (Optional[Union[float, Tuple[float, float]]]):
+        The number of seconds to wait for the server response.
+        Depending on the retry strategy, a request may be repeated
+        several times using the same timeout each time.
+
+        Can also be passed as a :func:`~google.cloud.requests.tuple()` (connect_timeout, read_timeout).
+        See :meth:`google.cloud.requests.Session.request()` documentation for details.
+"""
+        lines_got = lines_got.split("\n")
+        # Resolve over different regular expressions for different types of reference patterns.
+        lines_got, xrefs = _resolve_reference_in_module_summary(REF_PATTERN, lines_got)
+        lines_got, xrefs = _resolve_reference_in_module_summary(REF_PATTERN_LAST, lines_got)
```

Comment:
Anything to check for `xrefs`?

Reply:
Yes, we should check them. I'll add a test to check for `google.cloud` uids.